flask_project/campaign_manager/views.py

```diff
@@ -692,14 +697,16 @@ def find_attribution(map_url):
 
 @campaign_manager.route('/create', methods=['GET', 'POST'])
 def create_campaign():
+    from datetime import datetime
+    print('create_campaign')
+    print(datetime.now().time())
```

Comment:
do we need this print outside of a testing environment?

Reply:
we don't.